R/HDF5AnnData.R

```diff
@@ -359,7 +359,7 @@ HDF5AnnData <- R6::R6Class("HDF5AnnData", # nolint
       if (private$.close_on_finalize) {
         self$close()
       }
-      return(invisible(self))
+      return(invisible(self)) # nolint
```

Comment:
Removing the `return()` is probably better than disabling the linting but this should be fixed by #229

```suggestion
      invisible(self)
```